servers/cromwell/jobs/controllers/capabilities_controller.py

```diff
@@ -10,26 +10,36 @@ def get_capabilities():
     Returns:
         CapabilitiesResponse: Response containing this backend's capabilities
     """
-    capabilities = CapabilitiesResponse(
-        display_fields=[
-            DisplayField(field='status', display='Status'),
-            DisplayField(field='submission', display='Submitted'),
-            DisplayField(
-                field='labels.cromwell-workflow-name',
-                display='Workflow Name'),
-            DisplayField(
-                field='labels.cromwell-workflow-id', display='Workflow ID'),
-            DisplayField(field='labels.comment', display='Comment')
-        ],
-        common_labels=[
-            'cromwell-workflow-name', 'cromwell-workflow-id', 'comment'
-        ],
-        query_extensions=[])
-    if current_app.config['use_caas']:
-        capabilities.authentication = AuthenticationCapability(
-            is_required=True,
-            scopes=[
-                'https://www.googleapis.com/auth/userinfo.profile',
-                'https://www.googleapis.com/auth/userinfo.email'
-            ])
-    return capabilities
+    # Set default capabilities
+    capabilities = {
+        'display_fields': {
+            'status': 'Status',
+            'submission': 'Submitted',
+            'labels.cromwell-workflow-id': 'Workflow ID'
+        },
+        'common_labels': ['cromwell-workflow-id'],
+        'query_extensions': []
+    }
+
+    capabilities_config = current_app.config['capabilities_config']
+    if capabilities_config:
+        capabilities.update(capabilities_config)
+
+    display_fields, common_labels, query_extensions = [], set([]), set([])
+
+    # Construct display_fields
+    for field, display in capabilities['display_fields'].items():
+        display_fields.append(DisplayField(field=field, display=display))
+
+    # Construct common_labels, which is used by the query builder
+    common_labels.update(capabilities['common_labels'])
```

Comment:
Instead of setting `common_labels` to an empty set and updating its value, could we do `common_labels = set(capabilities['common_labels'])`? And also with `query_extensions` below?